test/regression/sql/transaction_errors.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE foo AS SELECT 'bar' AS t;
+BEGIN; SET duckdb.execution = true; SELECT t::integer AS t1 FROM foo; ROLLBACK;
+SET duckdb.execution = true;
+SELECT 1 FROM foo;
```

Comment:
missing newline at end of file

Reply:
I would expect `make format` to catch these - is it normal that it doesn't?

Reply:
We should probably add a script to check for that. `make format` only handles C, C++ and Python files so we need something different for all others (SQL, markdown, _schedule). If you install editorconfig it should do this on save though.

Reply:
Yep I did now, but I didn't go through all the files in this PR earlier (now I did :)